### PR TITLE
chore: Add RayanAbid.json

### PR DESCRIPTION
**Pull request by RayanAbid**
Added my profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1567"></a>

### CODE REVIEW

File: public/data/RayanAbid.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/rayanabid",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rayanDoesTech",
+      "icon": "twitter"
+    }
+  ],
```
Comment (kumarsonsoff3) [inline public/data/RayanAbid.json:22]: Kindly remove this Trailing comma from there! 🛠
```suggestion
  ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

File: public/data/RayanAbid.json (hunk 1 above)
Comment (RayanAbid) [reply public/data/RayanAbid.json:22]: Thank you. I was wondering what might be causing this.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by kumarsonsoff3 — APPROVED** [review]
Good to go! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**